core/api/src/domain/quiz/config.ts

```diff
@@ -1,6 +1,8 @@
+import { QuizQuestionId } from "./index.types"
+
+export const milliSecondsBetweenSections = 60 * 60 * 12 * 1000
```

Comment:
please move to quizzes config (also this is some of the reasons why having a `config` in domain is not good